### ISSUE TITLE
What does this mean with no math?

### ISSUE THREAD

**Issue by FG-CTRI** [LMLhub/ee_lecture_notes#17]
Explaining EE to my wife received the following comment: Can you explain what this means with no math? Great question. The message will be hear correctly if we can express it in the language of the various audience. The larger audience, the 99%, speaks no math.  If we do not frame the message in plain English, someone else will do it for us and usually with bad, intended or unintended, results.

What do you think of the following, no math, expression of EE for the 99%:

"Ergodic Economics shows that finance is based on a 350 years old, incomplete, and erroneous model. To provide recommendations the traditional model requires additional, external, and unreliable inputs such as psychological preferences.

The historical development of the traditional model created a succession of patches around this old, incomplete, and erroneous foundation. Departures or disagreements with the model are labeled as puzzles, paradoxes, anomalies, and irrational behavior.

Ergodic Economics fixes the erroneous foundation, works alongside the traditional model, and transcends its results. Ergodic Economics provides recommendations from inside the model, and resolves the puzzles, paradoxes, anomalies, and irrationalities."

**Comment by unbalancedparentheses** [comment]
Great idea. With Marcos we want to star a Frequently Asked Question. I think what you wrote is a good start. Would you like to help with the FAQ

**Comment by FG-CTRI** [comment]
Happy to help with the FAQ.

How do we start?

Note: To avoid the equivalent of the 1616 Papal Ban on Copernicus' 
ideas, we may want to emphasize how EE works alongside the standard 
model, Phase1/Phase2, lifts above the standard model to make it a 
special case, etc.

**Comment by unbalancedparentheses** [comment]
Sorry for the delay I am at a conference.

Excellent work on #18. Let's discuss it over there and use the small summary you wrote on this issue.
